### DISCUSSION TITLE
How to use Google Cloud storage

### DISCUSSION THREAD

**Discussion by Taiki0807** [pylixm/django-mdeditor#178] [Q&A]
@pylixm 

I am using Cloud storage but it is not saving. Can I specify my own image upload endpoint?

**Comment by pylixm** [comment]
Yes, you can refer to https://github.com/pylixm/django-mdeditor/pull/168

**Reply by Taiki0807** [reply]
@pylixm 

The following error occurs: 
"detail": "CSRF Failed: CSRF token missing."

One way to solve this is to create a callback to allow manipulation of strings containing form information?

https://github.com/Taiki0807/portfolio-backend/blob/1b1da907c979b3f01fb401b02bb85f3ef3f71cfe/config/urls.py#L16
